game_api.txt

```diff
@@ -83,6 +83,16 @@ in `bones.player_inventory_lists`.
 
 e.g. `table.insert(bones.player_inventory_lists, "backpack")`
 
+Additionally, callbacks can be registered to transfer items into the bones on death:
+
+`bones.public.register_transfer_inventory_to_bones_on_player_death(function(player){})`
```

Comment:
We don't do that here. Just `bones.name`. And please give it a shorter name.

Reply:
> And please give it a shorter name.
I can't come up with a short but descriptive name
would bones.register_dead_player_inv_management be short enough?